stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/MessageInputView.kt

```diff
@@ -527,6 +549,27 @@ public class MessageInputView : ConstraintLayout {
         }
     }
 
+    /**
+     * Creates an instance of [SuggestionListControllerListener].
+     *
+     * Used to modify attachment button visibility to discourage mixing
```

Comment:
I would rephrase the comment here.

```
Used to disable integration buttons, based on if commands or attachments are added to the message.
```

Discouraging makes sense but at the same time implies it can be broken and ignored. We shouldn't really _encourage_ such behavior and just be straight up with what this does.